tests/visualization_tests/test_optimization_history.py

```diff
@@ -209,11 +221,57 @@ def objective(trial: Trial) -> float:
     for x, study in zip(suggested_params, studies):
         study.enqueue_trial({"x": x})
         study.optimize(objective, n_trials=1)
-    figure = plot_optimization_history(studies, error_bar=True)
-
+    info_list = _get_optimization_history_info_list(
+        studies, target=None, target_name="Objective Value", error_bar=True
+    )
     mean = np.mean(suggested_params)
     std = np.std(suggested_params)
-
-    np.testing.assert_almost_equal(figure.data[0].y, mean)
-    np.testing.assert_almost_equal(figure.data[2].y, mean + std)
-    np.testing.assert_almost_equal(figure.data[3].y, mean - std)
+    assert info_list == [
+        _OptimizationHistoryInfo(
+            [0],
+            _ValuesInfo([mean], [std], "Objective Value"),
+            _ValuesInfo([mean], [std], "Best Value"),
```

Comment:
mypy raises the following errors:
> tests/visualization_tests/test_optimization_history.py:232: error: List item 0 has incompatible type "floating[Any]"; expected "float"
> tests/visualization_tests/test_optimization_history.py:233: error: List item 0 has incompatible type "floating[Any]"; expected "float"

Couldy you add `.item()` to convert `np.floating` to built-in `float`?

Reply:
Thanks! I fixed it.

Reply:
Thanks!